src/styles/blocks/main.scss

```diff
@@ -0,0 +1,8 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 938px;
```

Comment:
It is forbidden to set sizes and margins for blocks, but for block elements are allowed. The block should stretch to accommodate its children